### PR TITLE
[enterprise-4.10]GH#46935: Fix indentation in the sample remediation template YAML file

### PR DESCRIPTION
**Pull request by abhatt-rh**
Fixes https://github.com/openshift/openshift-docs/issues/46935: Fixes indentation in the sample remediation template YAML

Version(s):
OpenShift 4.10 only

Issue:
https://github.com/openshift/openshift-docs/issues/46935

Link to docs preview:
http://file.rdu.redhat.com/avbhatt/issue-46935/nodes/nodes/eco-poison-pill-operator.html#understanding-poison-pill-remediation-template-config_poison-pill-operator-remediate-nodes

### CODE REVIEW

**Comment by abhatt-rh** [comment]
Hi @prabinovRedhat @mshitrit,
Request you to review and approve this change for updated indentation to the template YAML file. Thanks!

**Comment by mshitrit** [comment]
Run at it through online yaml [validator](http://www.yamllint.com/), lgtm :+1:

**Comment by prabinovRedhat** [comment]
I think we need add one more indentation

**Comment by prabinovRedhat** [comment]
Screen Shot 2022-09-15 at 9 47 17

**Comment by prabinovRedhat** [comment]
@mshitrit wdyt?

**Comment by abhatt-rh** [comment]
> I think we need to add one more indentation, under `spec, template` etc (see below picture)

Hi @prabinovRedhat We have the indentation that you mention. Check the preview link. (I, too, referred the YAML from the web console and ran it in a YAML linter :) )

**Comment by mshitrit** [comment]
Good catch @prabinovRedhat !, I think you are right.
@abhatt-rh IIUC there needs to be two spaces and there is only one.

i.e currently we have:
```
apiVersion: poison-pill.medik8s.io/v1alpha1
kind: PoisonPillRemediationTemplate
metadata:
 creationTimestamp: "2022-03-02T08:02:40Z"
 generation: 1
 name: poison-pill-default-template
 namespace: openshift-operators
 resourceVersion: "596469"
 uid: 5d29e437-c485-48fa-ba9e-0354649afd31
spec:
 template:
  spec:
   remediationStrategy: NodeDeletion (1)
```
   
And what we should have is:

```
apiVersion: poison-pill.medik8s.io/v1alpha1
kind: PoisonPillRemediationTemplate
metadata: 
  creationTimestamp: "2022-03-02T08:02:40Z"
  generation: 1
  name: poison-pill-default-template
  namespace: openshift-operators
  resourceVersion: "596469"
  uid: 5d29e437-c485-48fa-ba9e-0354649afd31
spec: 
  template: 
    spec: 
      remediationStrategy: NodeDeletion (1)   
```

**Comment by abhatt-rh** [comment]
> 

Thank you, @prabinovRedhat and @mshitrit !! Fixed it with the additional space. Please approve if it looks good.
Wonder why it wasn't flagged by the YAML lint though.

**Comment by prabinovRedhat** [comment]
looks good. /lgtm

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Comment by abhatt-rh** [comment]
Thank you, @opayne1!
/label merge-review-needed

File: modules/eco-poison-pill-operator-about.adoc [hunk 1]
```diff
@@ -67,15 +67,15 @@ The `PoisonPillRemediationTemplate` CR resembles the following YAML file:
...
+  generation: 1
+  name: poison-pill-default-template
+  namespace: openshift-operators
+  resourceVersion: "596469"
+  uid: 5d29e437-c485-48fa-ba9e-0354649afd31
 spec:
-remediationStrategy: NodeDeletion <1>
+  template:
+    spec:
+      remediationStrategy: NodeDeletion (1)
```
Comment (bergerhoffer) [inline modules/eco-poison-pill-operator-about.adoc:79]: I'm surprised it works as is (at least in the preview), but the correct syntax for the callout here is `<1>`. Would you mind updating it? I'm afraid it's possible that the customer portal might not render it properly.

Comment (abhatt-rh) [reply modules/eco-poison-pill-operator-about.adoc]: Good catch, @bergerhoffer! I have updated it now.

**Review by bergerhoffer — COMMENTED** [review]
LGTM merging!

**Pull request merged**